kwokfixture/kwokfixture.go

```diff
@@ -0,0 +1,47 @@
+package kwokfixture
+
+import (
+	"fmt"
+	"os/exec"
+)
+
+const (
+	contextName      = "kwok"
+	clusterServerURL = "http://127.0.0.1:8080"
+)
+
+func runOcCommand(args ...string) error {
+	cmd := exec.Command("oc", args...)
+	if err := cmd.Run(); err != nil {
+		return fmt.Errorf("failed to run oc command: %w", err)
+	}
+	return nil
+}
+
```

Comment:
I think we could and should replace the code below with a simple fixed kubeconfig file committed into the repo.

Reply:
We already have another ticket for creating kubeconfig.

Reply:
so what is the point writing code we will throw away later?

Reply:
This is already there I am just moving it to different file for now